cli/lsp/documents.rs

```diff
@@ -186,21 +243,25 @@ impl SyntheticModule {
     }
   }
 }
-
 #[derive(Debug)]
-pub(crate) struct Document {
+struct DocumentInner {
   line_index: Arc<LineIndex>,
   maybe_language_id: Option<LanguageId>,
   maybe_lsp_version: Option<i32>,
   maybe_module:
     Option<Result<deno_graph::Module, deno_graph::ModuleGraphError>>,
   maybe_navigation_tree: Option<Arc<tsc::NavigationTree>>,
   maybe_warning: Option<String>,
-  source: SourceTextInfo,
+  text_info: SourceTextInfo,
   specifier: ModuleSpecifier,
   version: String,
 }
 
+#[derive(Debug, Clone)]
+pub(crate) struct Document {
+  inner: Arc<DocumentInner>,
+}
```

Comment:
```suggestion
pub(crate) struct Document(Arc<DocumentInner>);
```

I know it is less "explicit" but it is all just passthrough anyways.

Reply:
Done.